### PR TITLE
removing info log

### LINKED ISSUE #30394: [Filebeat] Reduce useless console log in filestream input
After switchd input type from log to filestream, there are repeat logs every 10s in filebeat console from 
https://github.com/elastic/beats/blob/main/filebeat/input/filestream/fswatch.go#L137
```
2022-01-26T09:06:45.287Z [file_watcher] filestream/fswatch.go:137 Start next scan
2022-01-26T09:06:55.286Z [file_watcher] filestream/fswatch.go:137 Start next scan
2022-01-26T09:07:05.287Z [file_watcher] filestream/fswatch.go:137 Start next scan
2022-01-26T09:07:15.286Z [file_watcher] filestream/fswatch.go:137 Start next scan
2022-01-26T09:07:25.286Z [file_watcher] filestream/fswatch.go:137 Start next scan
2022-01-26T09:07:35.286Z [file_watcher] filestream/fswatch.go:137 Start next scan
```
It seems meaningless, can it be removed from source code? Or at least can you change it's level to DEBUG or something else to avoid it appear in default setting.

### PR DESCRIPTION
**Pull request by solrac97gr**
## What does this PR do?
Remove the info log, because this were generating unnecessary logs.

## Why is it important?
That generate a big number of log who can distract of the real problem.

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have made corresponding change to the default configuration files
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

## Author's Checklist

- [ ] Check if it's also convenient just change the info to debug log

## How to test this PR locally
check in the watch logs there is not showing the "Start Next Scan" log

## Related issues

- Closes #30394

## Use cases

Scenario: Your Watch get a Event 
    When the Watcher have new Event
    Then the Watcher will not create a log with "Start Next Scan" in every event

### CODE REVIEW

**Comment by cmacknz** [comment]
> I already sign the Licence

@solrac97gr I checked I see the license signed with the hotmail address in your github profile, but the email for the commit is a gmail address. The CLA is looking at the commit email, not the PR. Either signing the CLA with your the gmail address in the commit or changing the commit to use your hotmail address will fix the check.

**Review by kvch — CHANGES_REQUESTED** [review]
Please do not remove this log message. It has helped me on many occasions to track down issues. Instead decrease the log level to debug.

**Comment by solrac97gr** [comment]
@cmacknz changing no problem

File: filebeat/input/filestream/fswatch.go [hunk 1]
```diff
@@ -133,7 +134,7 @@ func (w *fileWatcher) Run(ctx unison.Canceler) {
 }
 
 func (w *fileWatcher) watch(ctx unison.Canceler) {
-	w.log.Info("Start next scan")
+    w.log.Debug("Start next scan")
```
Comment (cmacknz) [inline filebeat/input/filestream/fswatch.go:137]: @kvch are you fine with making this a debug log instead of info?

**Comment by cmacknz** [comment]
> already try changing mail and signin with both mails but not result

I had to re-trigger the verification. It is signed now, and the CI checks can continue.

File: filebeat/input/filestream/fswatch.go (hunk 1 above)
Comment (kvch) [reply filebeat/input/filestream/fswatch.go:137]: Yes, thanks.

**Review by cmacknz — APPROVED** [review]

File: filebeat/input/filestream/fswatch.go (hunk 1 above)
Comment (cmacknz) [inline filebeat/input/filestream/fswatch.go:137]:
```suggestion
	w.log.Debug("Start next scan")
```

This needs to be a tab, not spaces. Let's see if I can fix this with a suggestion.

**Comment by zhiweiv** [comment]
Is it possible to cherry-pick this change to 7.17 branch?

**Comment by cmacknz** [comment]
@zhiweiv 7.17 cherry pick is here: https://github.com/elastic/beats/pull/32252, it will be a part of the next 7.17 patch release.

**Pull request merged**